### PR TITLE
[greynoise-266] Add ability to set Integration Reliability Value

### PR DESCRIPTION
**Pull request by bradchiappetta**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Adds the ability for users to define the "Integration Reliability" setting so that enrichment data reliability can be defined.

## Screenshots
Paste here any images that will help the reviewer

image

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
Hi @bradchiappetta, 
Thank you for contributing!
Apologies for the delay in reviewing your PR, we will review your code as soon as possible.

Thanks again.

**Review by israelpoli — CHANGES_REQUESTED** [review]
Hi @bradchiappetta 
Thank you for your contribution,

Please refer to the comments above,

The tests don't pass properly, at first glance, it appears that the input_data file needs a link change.

File: Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py [hunk 1]
```diff
@@ -941,11 +995,11 @@ def main() -> None:
             return_results(result)
 
         elif demisto.command() == "greynoise-ip-quick-check":
-            result = ip_quick_check_command(client, demisto.args())
+            result = ip_quick_check_command(client, demisto.args(), reliability)
```
[the remark this answers is not in the document]
Comment (bradchiappetta) [reply Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py:998]: Yes, correcting.

File: Packs/GreyNoise/ReleaseNotes/1_2_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### GreyNoise
+- Update IP enrichment commands to use Reliability option
```
[the remark this answers is not in the document]
Comment (bradchiappetta) [reply Packs/GreyNoise/ReleaseNotes/1_2_0.md:3]: Updating.

**Comment by israelpoli** [comment]
Hi @bradchiappetta 

Please take a look at the two failed jobs,
By clicking the `details` button, you can view the errors, and suggestions for solutions.

There are three errors:
1. Please change the `https://www.greynoise.io/viz/ip/71.6.135.131` link to the new link in the input_data file as the UT fails due to the change of the link.
2. Please make sure that the UT coverage is above 71.34, as this was the last coverage. (Adding UT to return the indicator in the `greynoise-riot` command, etc.)
3. Merge from master (Run ./Utils/git_pull_master_into_fork.sh or merge manually from upstream demisto content)

Let me know if you need help correcting the things above in Slack or email ipolishuk@paloaltonetworks.com.

**Thanks**

**Comment by bradchiappetta** [comment]
> 

@israelpoli : I've addressed two of the three items here, but having an issue with the code coverage one.  The CircleCI error is showing the issue is with the GreyNoise_Community integration, but you mention the GreyNoise integration.  I tried pushing what I thought was an update to fix the community one, but it didn't change the output.  Can you confirm which one actually has an issue?

**Comment by israelpoli** [comment]
> > 
> 
> @israelpoli : I've addressed two of the three items here, but having an issue with the code coverage one. The CircleCI error is showing the issue is with the GreyNoise_Community integration, but you mention the GreyNoise integration. I tried pushing what I thought was an update to fix the community one, but it didn't change the output. Can you confirm which one actually has an issue?

@bradchiappetta 
you are right,
The problem is with the GreyNoise_Community integration,
Because of the addition in the main function, you don't need to add UT to it,
I will consult with my team leader about this.

**Comment by israelpoli** [comment]
> > > 
> > 
> > 
> > @israelpoli : I've addressed two of the three items here, but having an issue with the code coverage one. The CircleCI error is showing the issue is with the GreyNoise_Community integration, but you mention the GreyNoise integration. I tried pushing what I thought was an update to fix the community one, but it didn't change the output. Can you confirm which one actually has an issue?
> 
> @bradchiappetta you are right, The problem is with the GreyNoise_Community integration, Because of the addition in the main function, you don't need to add UT to it, I will consult with my team leader about this.

Hi @bradchiappetta 
If you can add UT to the main function in GreyNoise_Community integration,
This is the only way to solve the problem
Thanks

**Comment by bradchiappetta** [comment]
> Details

@israelpoli : I think the latest commit gets things back where they need to be.  Hopefully, this is good to move forward now.  Thanks.

**Review by israelpoli — APPROVED** [review]
Good job

Thanks

**Pull request merged**